### ISSUE TITLE
java.lang.NullPointerException Attempt to invoke interface method 'retrofit2.Call

### ISSUE THREAD

**Issue by stantoncbradley** [applivery/applivery-android-sdk#18]
this error is crashing app. happens randomly, haven't been able to figure out what causes it
```
java.lang.NullPointerException Attempt to invoke interface method 'retrofit2.Call com.applivery.applvsdklib.network.api.AppliveryApiService.sendFeedback(com.applivery.applvsdklib.network.api.model.ApiFeedbackData)' on a null object reference 
    FeedbackRequest.java:65 com.applivery.applvsdklib.network.api.requests.FeedbackRequest.performRequest
    ServerRequest.java:31 com.applivery.applvsdklib.network.api.requests.ServerRequest.execute
    FeedbackInteractor.java:66 com.applivery.applvsdklib.domain.feedback.FeedbackInteractor.performRequest
    BaseInteractor.java:54 com.applivery.applvsdklib.domain.BaseInteractor.run
    ThreadPoolExecutor.java:1162 java.util.concurrent.ThreadPoolExecutor.runWorker
    ThreadPoolExecutor.java:636 java.util.concurrent.ThreadPoolExecutor$Worker.run
    Thread.java:764 java.lang.Thread.run
```

**Comment by CesarTrigo** [comment]
Hi @stantoncbradley,
Thank you for opening the issue, the team is looking into it and will come back to you ASAP.
Best,
C.

**Comment by mrebollob** [comment]
Fixed in b25cf2e1651c3afe7e6be6bc67f1ab8a75d04339

**Comment by chidozieugboma** [comment]
Please guys i am having the same issue how do i resolve this please help
